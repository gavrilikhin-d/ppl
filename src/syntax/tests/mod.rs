use crate::test_compilation_result;

test_compilation_result!(consume_greater);
test_compilation_result!(escaped_id);
test_compilation_result!(multiple_errors);
