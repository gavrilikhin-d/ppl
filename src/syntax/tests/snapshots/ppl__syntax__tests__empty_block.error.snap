---
source: src/syntax/tests/mod.rs
expression: err
---
Error: parser::empty_block

  × empty blocks are disallowed
   ╭─[empty_block.ppl:1:1]
 1 │ loop:
   · ──┬─
   ·   ╰── this item has empty block
   ╰────
